### PR TITLE
ini_file: fix regression reported in #2578

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fixes the regression reported by @jbronn in #2578.

The first commit basically adds a test that fails. Let's see whether CI actually fails with it, so we can try to understand why we didn't notice this earlier.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
ini_file

### CODE REVIEW

**Comment by felixfontein** [comment]
@jbronn I *think* the problem is that the tests didn't have anything that's not ASCII, and the failure only happens once you have something that's not ASCII.

**Comment by felixfontein** [comment]
ready_for_review

**Review by quidame — APPROVED** [review]
Hi @felixfontein thanks for fixing this !

File: plugins/modules/files/ini_file.py [hunk 1]
```diff
@@ -112,6 +112,7 @@
 import traceback
 
 from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_bytes, to_text
```
[the remark this answers is not in the document]
Comment (felixfontein) [reply plugins/modules/files/ini_file.py]: I think we should update all these imports, I think they happen in a lot other places as well :)

**Review by quidame — COMMENTED** [review]
LGTM

**Comment by jpmens** [comment]
shipit

**Comment by felixfontein** [comment]
@quidame @jpmens thanks for reviewing this! :)

**Pull request merged**